tdx-tdcall/src/tdx.rs

```diff
@@ -345,10 +345,10 @@ pub fn tdvmcall_cpuid(eax: u32, ecx: u32) -> CpuIdInfo {
     }
 
     CpuIdInfo {
-        eax: args.r12 as u32,
-        ebx: args.r13 as u32,
-        ecx: args.r14 as u32,
-        edx: args.r15 as u32,
+        eax: (args.r12 & 0xffffffff) as u32,
```

Comment:
Nit: Might be worth using an underscore for clarity (here and elsewhere):

```suggestion
        eax: (args.r12 & 0xffff_ffff) as u32,
```

Reply:
Thanks! Fixed